tendrl_node_agent/Manager.py

```diff
@@ -0,0 +1,46 @@
+import signal
+import gevent.event
+import greenlet
+import gevent.greenlet
+from tendrl_node_agent.monitoring.AlertSocket import AlertsManager
+from tendrl_node_agent.monitoring.MonitoringConfigManager import CollectdConfigManager
+
+class TendrlManager():
+    def __init__(self):
+        self._complete = gevent.event.Event()
+        self.collectd_config_manager = CollectdConfigManager()
+        self.collectd_config_manager.start()
```

Comment:
define a constant for port and localhost and use those instead of hardcodng the values